commit f2ac53cd4a137ed263deb9ae22ee00ef2b32fbb3 (refs/changes/47/547/1)
Author: Kris Shannon <k.shannon@amaze.com.au>
Date:   2016-09-29T20:06:44+10:00 (3 years ago)
    
    Add a --keyPath default cli option to specify file containing the SSH private key

diff --git a/lib/create_client.js b/lib/create_client.js
index 86b753b..3342485 100644
--- a/lib/create_client.js
+++ b/lib/create_client.js
@@ -56,6 +56,13 @@ var DEFAULT_OPTIONS = [
         help: 'SSH key fingerprint',
         helpArg: 'FINGERPRINT'
     },
+    {
+        names: ['keyPath'],
+        type: 'string',
+        env: 'MANTA_KEY_PATH',
+        help: 'SSH private key file',
+        helpArg: 'FILE'
+    },
     {
         names: ['url', 'u'],
         type: 'string',
